examples/models/local_hugging_face/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Local Hugging Face Models</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+	<h1>Local Hugging Face Models</h1>
+	<form id="completions-form" action="http://localhost:8000/completions" method="POST">
+		<label for="prompt">Prompt:</label>
+		<br><br>
+		<textarea id="prompt" name="prompt" required></textarea><br>
+		<br><br>
+		<button type="submit">Generate Completions</button>
+	</form>
+
+	<br><br>
+	<div id="completions-results"></div>
+
+	<script>
+		// Get the form element and the model and prompt input elements
+		const form = document.getElementById('completions-form');
+		const promptInput = document.getElementById('prompt');
+
+		async function handleSubmit(event) {
+		  // Prevent the default behavior of the form submission (i.e. page reload)
+		  event.preventDefault();
+
+		  // Get the model and prompt values from the input elements
+		  const promptValue = promptInput.value;
+
+		  // Get the active model from the window.ai API 
+		  await window.ai.getCurrentModel();
```

Comment:
maybe store this as a variable and include it in the output? it's being ignored now

Reply:
Yep, added it to the HTML output